data/discovery/field/citizens-advice-bureau-membership-number.yaml

```diff
@@ -0,0 +1,6 @@
+cardinality: 'n'
+datatype: string
+field: citizens-advice-bureau-membership-number
+phase: discovery
+register: citizens-advice-bureau-membership-number
```

Comment:
I don't think this needs its own register, but perhaps it should be the unique ID of the `citizens-advice-bureau` register.